pkg/handler/api.go

```diff
@@ -96,10 +99,13 @@ func createLink(c echo.Context, offset int) string {
 	req := c.Request()
 	q := req.URL.Query()
 	page := ParsePagination(c)
+	filters := ParseFilters(c)
 
 	q.Set("limit", strconv.Itoa(page.Limit))
 	q.Set("offset", strconv.Itoa(offset))
-
+	q.Set("search", filters.Search)
+	q.Set("arch", filters.Arch)
+	q.Set("version", filters.Version)
```

Comment:
should these just be omitted if they are blank?

Reply:
Yes. Will do some if statement magic.